### PR TITLE
Chatbot hotfix

### PR DESCRIPTION
**Pull request by aliabid94**
Hotfix involving css parameter and output state, which were causing issues

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <link rel='stylesheet' href='build/themes.css'>
 
   <link rel="stylesheet" href="build/global.css">
+  <style></style>
```
Comment (abidlabs) [inline frontend/public/index.html:11]: Why?

**Review by pngwn — APPROVED** [review]

**Comment by abidlabs** [comment]
Looks good to me

File: frontend/public/index.html (hunk 1 above)
Comment (aliabid94) [reply frontend/public/index.html:11]: The custom css will be dumped into this element

**Review by abidlabs — APPROVED** [review]

File: frontend/public/index.html (hunk 1 above)
Comment (abidlabs) [reply frontend/public/index.html:11]: Gotcha

Comment (aliabid94) [reply frontend/public/index.html:11]: actually might be better to create a new element dunamically, you're right

**Pull request merged**